distribution/docs/src/main/resources/content/_coreConcepts/authentication-standards.adoc

```diff
@@ -0,0 +1,25 @@
+:type: subCoreConcept
+:status: published
+:title: Authentication Standards Provided by ${ddf-branding}
+:parent: Standards Supported by ${branding}
+:order: 50
+
+.{title}
+[cols="2,1" options="header"]
+|===
+|Standard
+|Support Status
+
+|http://www.oasis-pki.org/resources/techstandards/[Public Key Infrastructure (PKI)]
+|Supported
+
+|https://www.ietf.org/rfc/rfc2617.txt[Basic Authentication]
+|Supported
+
+|https://www.oasis-open.org/standards#samlv2.0[SAML]
```

Comment:
❓  since we link to specific version should we state the version number? It may be fine as it is so just a question since the link takes you there. This seems better from a maintenance perspective for sure.

Reply:
Maintenance is important. We do list versions in other places where there are versions we don't support, whereas I believe in this case we do support all available ones.